src/raster/r.damflood/r.damflood.html

```diff
@@ -1,140 +1,197 @@
 <h2>DESCRIPTION</h2>
 
 
-<em><b>r.damflood</b></em> - The definition of flooding areas is of considerable importance for both the risk analysis and the emergency management.
-This command, in particular, is an embedded GRASS GIS hydrodynamic 2D model that allows to obtain flooding area due to a failure 
-of a dam, given the geometry of the reservoir and of the downstream area, the initial conditions and the dam breach geometry.
+<em><b>r.damflood</b></em> - The definition of flooding areas is of considerable
+importance for both the risk analysis and the emergency management.
+This module, in particular, is an embedded GRASS GIS hydrodynamic 2D model
+that allows to obtain flooding areas due to a failure of a dam, given 
+the geometry of the reservoir and of the downstream area, 
+the initial conditions and the dam breach geometry.
 <br>
 
-The numerical model solves the conservative form of the shallow water equations (SWE) using a finite volume method (FVM); 
-the intercell flux is computed by the "upwind method and the water-level gradient is evaluated by weighted 
-average of both upwind and downwind gradient. Additional details of the specific numerical scheme adopted in the model are 
+The numerical model solves the conservative form of the shallow water equations
+(SWE) using a finite volume method (FVM);
+the intercell flux is computed by the "upwind method and the water-level
+gradient is evaluated by weighted average of both upwind and downwind gradient. 
+Additional details of the specific numerical scheme adopted in the model are
```

Comment:
I wasn't able to find what the quotes were referring to, thus where the quote should end.

Reply:
I just removed the superfluous double quote